src/middlewared/middlewared/alembic/versions/12.0/2020-04-16_07-59_truecommand.py

```diff
@@ -0,0 +1,38 @@
+"""
+TrueCommand Service Model
+
+Revision ID: 58339783792c
+Revises: 171f5b91c36e
+Create Date: 2020-04-06 07:59:08.663553+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import middlewared.sqlalchemy as m_sa
```

Comment:
You should never import anything that can alter it's behavior in migrations, they are meant to be written once and then never changed.